src/kaocha/api.clj

```diff
@@ -99,12 +99,20 @@
           (with-bindings (config/binding-map config)
             (let [config (resolve-reporter config)]
               (let [test-plan (test-plan config)]
+                
                 (when-not (some #(or (hierarchy/leaf? %)
                                      (::testable/load-error %))
                                 (testable/test-seq test-plan))
-                  (output/warn (str "No tests were found, make sure :test-paths and "
-                                    ":ns-patterns are configured correctly in tests.edn."))
+                  (if (not (zero? (count (filter ::testable/skip (testable/test-seq-with-skipped test-plan)))))
+                    (output/warn (str "No tests were found and some tests were"
```

Comment:
This sounds like a contradiction, if "no tests were found" how can it be that "some of them are skipped"?

Maybe something like: "All ${count} tests were skipped. This could be due to incorrect `focus` or `skip` filters, or due to misspellings in your kaocha configuration."